reseq_ros2/reseq_ros2/constants.py

```diff
@@ -24,6 +29,26 @@ class ReseQTopic(NamedTuple):
     direction: Direction
     data_type: type
 
+class ReseQState(NamedTuple):
+    """
+    Data structure for JointState operations
+    """
+    topic: str
+    states: list[str]
+    state_type: StateType
+
+states = (
+    ReseQState("motor/feedback", ["left_front_wheel", "left_back_wheel", "right_front_wheel", "right_back_wheel"], StateType.MOTOR_FEEDBACK),
```

Comment:
These are not used, should we remove them?

Reply:
Do you mean line 41? I think it is used to create subscribers on the `motor/feedback` topics. I guess we don't need the `ReseQState` type, though. If you want to simplify the `create_subs` method, feel free to do so

Reply:
Right, we can leave it as it is for now